dev-tools/audius-compose

```diff
@@ -333,12 +321,6 @@ def up(
 
     AAO_DIR = pathlib.Path(os.getenv("AAO_DIR", protocol_dir / "../anti-abuse-oracle"))
 
-    # TODO: Consider removing /etc/hosts change on audius-compose down
-    hosts_file = pathlib.Path("/etc/hosts")
-    if LOCALHOSTS not in hosts_file.read_text():
-        print("Adding development hosts to /etc/hosts")
-        hosts_file.write_text(hosts_file.read_text() + "\n" + LOCALHOSTS + "\n")
-
```

Comment:
We need the code to log something out here instead, I think that might be the better approach

Reply:
synced offline